src/qr.rs

```diff
@@ -649,6 +683,15 @@ pub async fn set_config_from_qr(context: &Context, qr: &str) -> Result<()> {
                 .set_config_internal(Config::WebrtcInstance, Some(&instance_pattern))
                 .await?;
         }
+        Qr::Socks5Proxy { host, port } => {
+            context.set_config(Config::Socks5Host, Some(&host)).await?;
+            context
+                .set_config_u32(Config::Socks5Port, port as u32)
+                .await?;
+            context.set_config(Config::Socks5User, None).await?;
```

Comment:
User and password need to be reset first, otherwise if the app crashes before this, it will reuse old user/password with the new proxy

Reply:
maybe even disable Socks5 before changing config and only enable it when everything is set?